torchdynamo/optimizations/backends.py

```diff
@@ -582,3 +583,24 @@ def exec_tvm(*args):
         return [to_torch_tensor(m.get_output(i)) for i in range(m.get_num_outputs())]
 
     return exec_tvm
+
+try:
+    import lazy_tensor_core
+    from lazy_tensor_core import _LAZYC
+    # hopefully changing this line to sth like _ltc_init_xla_backend in future
+    # will enable XLA
+    _LAZYC._ltc_init_ts_backend()
```

Comment:
Can we do the imports/setup inside the backend function scope?  This will mean when using other backends we won't need to load extra stuff into memory.

You could follow the same pattern as [_init_tensorflow](https://github.com/jansel/torchdynamo/blob/main/torchdynamo/optimizations/backends.py#L217) to make it happen exactly once.